keel-core/src/main/kotlin/com/netflix/spinnaker/keel/logging/TracingSupport.kt

```diff
@@ -15,6 +15,8 @@ class TracingSupport {
   companion object {
     const val X_SPINNAKER_RESOURCE_ID = "X-SPINNAKER-RESOURCE-ID"
 
+    val clearMDC: MDCContext = MDCContext(emptyMap())
```

Comment:
I think this should be `emptyMDC` - when I read it as a parameter I thought it was a function that cleared the mdc context.

Actually, I think any wording for this - "empty", "clean", "clear" would have given me the same thought. Maybe `blankMDC`? Change if you think that is more helpful or clear, if you don't think it adds anything feel free to ignore.

Reply:
Renamed to `blankMDC`.